src/error.rs

```diff
@@ -25,10 +26,10 @@ pub enum Error {
     BufferTooShort(usize, &'static str),
 }
 
-#[cfg(feature = "std")]
 impl error::Error for Error {
     fn source(&self) -> Option<&(dyn error::Error + 'static)> {
         match *self {
+            #[cfg(feature = "std")]
```

Comment:
This is interesting, normally adding a cfg on a variant arm is a breaking change but since this entire impl had the cfg before I don’t think this can in principle break anyone ? Eg this change should be a backwards compatible change and hence minor update